lib/css_parser/rule_set.rb

```diff
@@ -457,6 +510,8 @@ def create_list_style_shorthand! # :nodoc:
 
   private
 
+    attr_accessor :declarations
```

Comment:
do we need a setter ?
```suggestion
    attr_reader :declarations
```

Reply:
ah nvm, wrong class